edward2/tracers.py

```diff
@@ -30,6 +30,8 @@
 from edward2.trace import trace
 from edward2.trace import traceable
 
+import tensorflow_probability as tfp  # TODO(davmre): probably can't do this?
```

Comment:
Is this for the set seed tracer? If so, you should move it to a new edward2/tensorflow/tracers.py module.